.github/CONTRIBUTING.md

```diff
@@ -1,3 +1,24 @@
+<!-- TOC -->
```

Comment:
Not needed, get's auto-generated by GitHub here:

![Screenshot 2023-03-27 at 1 51 14 PM](https://user-images.githubusercontent.com/4616906/228025299-c07bb877-2109-4023-8b3d-be1452956077.png)

(Oddly on the opposite side of the screen as the README one)